README.md

```diff
@@ -59,6 +59,12 @@ In order to use the application's chat functionality, the connected wallet must
 
 The application utilizes the `xmtp-js` [Conversations](https://github.com/xmtp/xmtp-js#conversations) abstraction to list the available conversations for a connected wallet and to listen for or create new conversations. For each convesation, it gets existing messages and listens for or creates new messages. Conversations and messages are kept in a lightweight store and made available through **XmtpContext** alongside the client and its methods.
 
+### XMTP Network
+
+By default the chat app connects to the XMTP `dev` network, which is designed for development. The `dev` network makes no guarantees of messsage persistence, and is periodically purged of all messages.
+
+To connect to the `production` network, please set the following environment variable `NEXT_EXPOSED_XMTP_ENVIRONMENT=production`. The production network persists messages long-term and is intended for publicly depoloyed production environments.
```

Comment:
Should this be `NEXT_PUBLIC_XMTP_ENVIRONMENT`?

Reply:
Oops. Good catch. My mind went to Retool